src/js/components/explorer/landing/ExplorerLanding.jsx

```diff
@@ -35,13 +37,13 @@ export default class ExplorerLanding extends React.Component {
             content = (
                 <div className="explorer-description__content">
                     <p>
-                        Drawing on agencies&apos; financial spending data known as <a href="#/explorer/?glossary=obligation">Obligations</a>, this tool provides an interactive way to explore federal spending from top to bottom. Use this tool to get a better sense of how Congress distributes funding to agencies and how agencies spend that funding on activities that fulfill their missions &mdash; in short, how this funding powers the work that agencies are known for.
+                        Drawing on agencies&apos; financial data, this tool provides an interactive way to explore federal spending (known as Obligations  <a href="#/explorer/?glossary=obligation"><Icons.Glossary /></a>) from top to bottom. Use this tool to get a better sense of how Congress distributes funding to agencies and how agencies spend that funding on activities that fulfill their missions &mdash; in short, how this funding powers the work that agencies are known for.
```

Comment:
Can you add alt tags to the icons so the links are meaningful to screenreader users? These icon components accept an `alt` prop.

Meaningful links are links whose labels indicate where they lead when taken out of context, for example: `Definition of Obligation` rather than `click here`.